client/src/hand/hand.view.js

```diff
@@ -18,9 +18,5 @@ export default function Hand({ cards }) {
 }
 
 Hand.propTypes = {
-    cards: PropTypes.arrayOf(PropTypes.shape({
-        suit: PropTypes.string,
-        cardValue: PropTypes.string,
-        id: PropTypes.number,
-    })),
+    cards: PropTypes.arrayOf(PropTypes.string),
```

Comment:
not coherent with props in component Card
If it is a string like "AH", then it should not be named cards